apps/admin/src/app-pages/dashboard/ui/DashboardPage.tsx

```
@@ -0,0 +1,848 @@
+'use client';
+
+import { Input } from '@surf/ui/input';
+import { useMemo, useState, type ReactNode } from 'react';
+import type {
+  ChurnUserResponse,
+  DashboardResponse,
+  FunnelResponse,
+  KpiDailyResponse,
+  NoticeSampleRowResponse,
+} from '@/features/dashboard/api/types';
+import { useDashboardDateRange } from '@/features/dashboard/model/useDashboardDateRange';
```

ESLint

[error] Unable to resolve path to module '`@/features/dashboard/model/useDashboardDateRange`'.

(import/no-unresolved)